ruby/vampires.rb

```diff
@@ -0,0 +1,128 @@
+# How many times do we need to fill out the questionnaire? 
+puts "How many employees are you processing right now?"
+employees = gets.chomp
+employees = employees.to_i.to_s.to_i
+while employees == 0 || employees == nil
+	puts "I'm sorry, could you try that again? Enter an integer greater than 0."
+	employees = gets.chomp
+	employees = employees.to_i.to_s.to_i
```

Comment:
`.to_i.to_s.to_i` why would you need to change from a string to an integer to a string to an integer?